elements/lisk-p2p/src/utils.ts

```diff
@@ -200,3 +201,6 @@ export const getBucket = (options: {
 
 	return hash(bucketBytes).readUInt32BE(0) % secondMod;
 };
+
+export const constructPeerIdFromPeerInfo = (peerInfo: P2PPeerInfo): string =>
```

Comment:
This method is duplicated. `peer/base.ts` is also exporting the same one.

Reply:
Yes I agree, I want to take it out and only use it from utils. I will remove it from the `peer/base.ts`